mtags-interfaces/src/main/java/scala/meta/pc/PresentationCompiler.java

```diff
@@ -161,4 +200,11 @@
      * detail that the compiler is `null` when it has been reset.
      */
     public abstract boolean isLoaded();
+
+    protected CompletionList emptyCompletion(){
```

Comment:
It was not necessary, but since it was useful to have in both PresentationCompiler implementations I thought it would good to have it here. Especially since this is java code we are interacting with.

I will put it into a separate object.

Reply:
It's here: https://github.com/scalameta/metals/pull/1523/files#diff-8d0d759de71bb32f0a0f4734991ae91dR1-R14